client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -73,13 +73,49 @@ class CustomContentTypes extends Component {
 				>
 					{ label }
 				</CompactFormToggle>
+
+				{ this.renderPostsPerPageField( name, label, ! fields[ name ] ) }
+
 				<FormSettingExplanation isIndented>
 					{ description }
 				</FormSettingExplanation>
 			</div>
 		);
 	}
 
+	renderPostsPerPageField( fieldName, postTypeLabel, isDisabled = false ) {
```

Comment:
`isDisabled` could be computed inside this function rather than being passed in?